synapse/lib/modelrev.py

```diff
@@ -245,6 +248,371 @@ async def save():
             if nodeedits:
                 await save()
 
+    def typeHasStortype(self, typedef, stortype):
+
```

Comment:
Please add some small (one liner is fine) doc strings to these methods just to state intended use case

Reply:
Added docstrings and a few comments